gap/oper.gi

```diff
@@ -309,6 +309,36 @@ function(D, u, v)
   return D;
 end);
 
+InstallMethod(DigraphContractEdge,
+"for a mutable digraph by out-neighbours and two positive integers",
+[IsMutableDigraph and IsDigraphByOutNeighboursRep, IsPosInt, IsPosInt],
+function(D, src, ran)
+  local V, P;
+  if IsMultiDigraph(D) then
+    ErrorNoReturn("the 1st argument <D> must be a digraph with no multiple ",
+                  "edges,");
+  elif not (IsPosInt(src) and src in DigraphVertices(D)) then
+    ErrorNoReturn("the 2nd argument <src> must be a vertex of the ",
+                  "digraph <D> that is the 1st argument,");
+  elif not (IsPosInt(ran) and ran in DigraphVertices(D)) then
+    ErrorNoReturn("the 3rd argument <ran> must be a vertex of the ",
+                  "digraph <D> that is the 1st argument,");
+  elif not ([src, ran] in DigraphEdges(D)) then
+    ErrorNoReturn("there must be an edge between ", src, " and ", ran);
```

Comment:
I think this should be more precise (currently is doesn't explicitly say which direction the edge should be)
```suggestion
    ErrorNoReturn("[<src>, <ran>] must be an edge of the digraph <D> ",
                  "(the 2nd, 3rd, and 1st arguments, respectively),");
```